execution/executor-benchmark/src/db_generator.rs

```diff
@@ -35,15 +29,31 @@ pub fn run(
     // create if not exists
     fs::create_dir_all(db_dir.as_ref()).unwrap();
 
-    let (config, genesis_key) = aptos_genesis::test_utils::test_config();
+    bootstrap_with_genesis(&db_dir);
+
+    println!("=============FINISHED DB CREATION =============");
+    println!("DB dir: {}", db_dir.as_ref().display());
```

Comment:
?? you want these printed after calling `add_accounts()`, I assume?

Reply:
I was planning to say "finish empty db creation". 

Changed it a little bit to make it more clear now.